web-console/knikubevirt/knikubevirt.md

```diff
@@ -0,0 +1,31 @@
+## KNI and KubeVirt Designs
+Access design documentation specific to the KNI and KubeVirt features in OpenShift.
+
+### OpenShift 4.2
+- Cluster Dashboard
+- [VM List](http://openshift.github.io/openshift-origin-design/web-console/knikubevirt/vm-list/vm-list)
+- [VM Templates](http://openshift.github.io/openshift-origin-design/web-console/knikubevirt/vm-templates/vm-templates)
+- [VM Details](http://openshift.github.io/openshift-origin-design/web-console/knikubevirt/vm-details/vm-details)
+
+### Future Designs
+
+#### KNI (Kubernetes Native Infrastructure/OpenShift on Bare Metal)
+- Storage Dashboard
+- Add Bare Metal Host
+- Remove Bare Metal Host
+- Update Bare Metal Cluster
+
+#### KubeVirt (Virtualization in OpenShift)
+- [Clone VM](http://openshift.github.io/openshift-origin-design/web-console/knikubevirt/clone-vm/clone-vm)
```

Comment:
I think this should go to 4.2

Reply:
Thanks, I will move it! One note, Sam mentioned we will not have any toast notifications for 4.2 so we should update this notification to inline until we can move to toasts in 4.3.

Reply:
Good point. Here is a [follow-up issue](https://github.com/openshift/openshift-origin-design/issues/215) where I think it just makes sense to remove the toast for now.

Reply:
Okay cool, are you good with me hiding that toast screenshot from the design doc for now?

Reply:
@beanh66 Fine by me! What do you think @matthewcarleton ?